### PR TITLE
Upgrade gatling version to 3.9.5

### PR DESCRIPTION
**Pull request by gold-kou**
### Description
- Upgrade gatling version to 3.9.5
- Upgrade java version to 17
- Add rm option to use gatling.sh

I checked this change works correctly at my local by following commands.

```
$ make kind-create
$ kubectl config use-context kind-gatling-cluster 
$ make build 
$ make install-crd 
$ make kind-deploy 
$ make kind-sample-deploy
```

Also, I confirmed that Gatling 3.2.1 works fine with `rm` option. This means whether upgrading gatling version or not is up to gatling-operator users.

### Checklist

_Please check if applicable_

- [ ] Tests have been added (if applicable, ie. when operator codes are added or modified)
- [ ] Relevant docs have been added or modified (if applicable, ie. when new features are added or current features are modified)

Relevant issue https://github.com/st-tech/gatling-operator/issues/84

### CODE REVIEW

File: gatling/Dockerfile [hunk 1]
```diff
@@ -1,24 +1,18 @@
...
 # gating version
-ENV GATLING_VERSION 3.2.1
+ENV GATLING_VERSION 3.9.5
 
 # create directory for gatling install
 RUN mkdir -p gatling
 
 # install gatling
-RUN apk add --update wget bash libc6-compat && \
+RUN apt-get update && apt install -y wget bash libc6-dev unzip && \
```
[the remark this answers is not in the document]
Comment (R-HNF) [reply gatling/Dockerfile:15]: I think `libc6` is required in Alpine, but I don't think it's necessary in Debian.

**Comment by gold-kou** [comment]
@R-HNF 
Thank you for all your comments.
I fixed them.

https://github.com/st-tech/gatling-operator/pull/96/commits/1bc64dfc92ab78a06dc5b93e57a4ba6fcd8b03a6

**Review by R-HNF — APPROVED** [review]
Thanks for your contribution! 👍🎉

**Comment by gold-kou** [comment]
@R-HNF 
I deleted unnecessary `libc6-dev` .

https://github.com/st-tech/gatling-operator/pull/96/commits/da6bc34fb826bd9ba377cfcf1cc2e44324262fa4

**Review by itiB — APPROVED** [review]
Great works:+1:
I Checked Gatlings was works in 3.9.5 version and it does not have some errors.
Looks nice

```
❯ make kind-sample-deploy
Cluster already exists
cd gatling && docker build -t gatling:20231003-214045 .
[+] Building 2.6s (13/13) FINISHED
...
 => => writing image sha256:8f183c800bbf4e865aadc9eacc6ded2bd84e16bdc7b31083cc08dc44af20b159                                                                                                                         0.0s
 => => naming to docker.io/library/gatling:20231003-214045                                                                                                                                                           0.0s
Loading sample image into kind
kind load docker-image gatling:20231003-214045 --name "gatling-cluster" -v 1
Image with ID: sha256:8f183c800bbf4e865aadc9eacc6ded2bd84e16bdc7b31083cc08dc44af20b159 already present on the node gatling-cluster-worker but is missing the tag docker.io/library/gatling:20231003-214045. re-tagging...
Image with ID: sha256:8f183c800bbf4e865aadc9eacc6ded2bd84e16bdc7b31083cc08dc44af20b159 already present on the node gatling-cluster-control-plane but is missing the tag docker.io/library/gatling:20231003-214045. re-tagging...
~/github.com/gold-kou/gatling-operator/bin/kustomize build config/samples | sed -e "s,^\([[:space:]]*gatlingImage: \).*,\1gatling:20231003-214045,g" | kubectl apply -f -
serviceaccount/gatling-operator-worker unchanged
role.rbac.authorization.k8s.io/pod-reader unchanged
rolebinding.rbac.authorization.k8s.io/read-pods configured
secret/gatling-notification-slack-secrets unchanged
gatling.gatling-operator.tech.zozo.com/gatling-sample01 created

❯ k get pods,gatling
NAME                                READY   STATUS    RESTARTS   AGE
pod/gatling-sample01-runner-q5jnb   1/1     Running   0          12s
pod/gatling-sample01-runner-t5lhw   1/1     Running   0          12s
pod/gatling-sample01-runner-x8frl   1/1     Running   0          12s

NAME                                                      RUNNED   REPORTED   NOTIFIED   REPORTURL   AGE
gatling.gatling-operator.tech.zozo.com/gatling-sample01   0/3                                        13s

❯ k exec -it pod/gatling-sample01-runner-q5jnb -- /bin/sh
Defaulted container "gatling-runner" out of: gatling-runner, gatling-waiter (init)
# echo $GATLING_VERSION
3.9.5
```

**Pull request merged**